static/js/interactive-map/NewMap.js

```diff
@@ -248,7 +313,9 @@ class NewMap extends ANSWERS.Component {
    * @param {Number} index The index of the entity in the result list ordering
    */
   buildPin(pinOptions, entity, index) {
+    const id = 'js-yl-' + entity.profile.meta.id;
```

Comment:
for my own edification, what `yl` here for?

Reply:
Just asked Consulting! This stands for yext locator, to differentiate from their other uses of some js `id` on the page.